test/test_tool_dwp_tools.py

```diff
@@ -2,11 +2,12 @@
 import os.path
 
 from gistools.utils.collection import MemCollection
-from gistools.tools.dwp_tools import get_haakselijnen_on_points_on_line, flip_lines
+from gistools.tools.dwp_tools import get_haakselijnen_on_points_on_line, flip_lines, get_leggerprofiel
 
 
 test_data_dir = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
 
+# TODO: uitzoeken hoe OrderedDict goed af te dwingen in genereren van dict en in AssertDictEqual
```

Comment:
Is deze opmerking relevant voor deze testfile (volgorde maakt overigens niet uit, zie test fix verderop in PR)